### DISCUSSION TITLE
Which license typ of third party libs be usefull for axmol?

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#450] [Q&A] [answered]
I want add a third Open Source 2D Physics Lib to adxe but:
https://github.com/JavierDega/pip has license typ: GNU General Public License v3.0

Which license typ of third party libs be usefull for adxe?

**Comment by halx99** [comment, marked as answer]
The MIT, apache should be ok
Note: Don't add GPLv3 to adxe, the GPLv3 doesn't allow use close source Commercial project
